flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/AvroSchemaConvertor.java

```diff
@@ -200,4 +241,204 @@ private static DataType handleLogicalTypeSchema(Schema fieldSchema) {
         }
         return null;
     }
+
+    /**
+     * Converts Flink SQL {@link org.apache.flink.table.types.logical.LogicalType} (can be nested)
+     * into an Avro schema.
+     *
+     * <p>Use "org.apache.flink.avro.generated.record" as the type name.
+     *
+     * @param schema the schema type, usually it should be the top level record type, e.g. not a
+     *     nested type
+     * @return Avro's {@link Schema} matching this logical type.
+     */
+    public static Schema convertToSchema(org.apache.flink.table.types.logical.LogicalType schema) {
```

Comment:
Why not import `LogicalType` instead of using the fully qualified name?

Reply:
We also use `org.apache.avro.LogicalType` in the same file.

Reply:
Fixed.